src/types/email.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongEmail = StrongType<string>;
 
-export function makeEmail(fallback: string, initial?: string | null): StrongEmail {
+export function makeEmail(fallback: string, initial: string | null | undefined): StrongEmail {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.